tembo-cli/src/cmd/apply.rs

```diff
@@ -41,28 +43,34 @@ const POSTGRESCONF_NAME: &str = "postgres.conf";
 
 /// Deploys a tembo.toml file
 #[derive(Args)]
-pub struct ApplyCommand {}
+pub struct ApplyCommand {
+    #[clap(long, short = 'm')]
+    pub merge: Option<String>,
+}
 
-pub fn execute(verbose: bool) -> Result<(), anyhow::Error> {
+pub fn execute(verbose: bool, _merge_path: Option<String>) -> Result<(), anyhow::Error> {
     info!("Running validation!");
     super::validate::execute(verbose)?;
     info!("Validation completed!");
 
     let env = get_current_context()?;
 
+    let instance_settings = get_instance_settings(_merge_path.clone())?;
+    println!("Instance settings: {:?}", instance_settings);
```

Comment:
You can add a println in the `get_instance_settings` function towards the end saying overlay file merged successfully.  I don't think you need to print the values but I'm fine keeping it if you think it will be helpful and just move it to `get_instance_settings` function.